docs/conf.py

```diff
@@ -108,6 +109,38 @@
 plot_working_directory = "."
 plot_html_show_source_link = False
 
+# ----------------------------------------------------------------------------------
+# Redirects
+# ----------------------------------------------------------------------------------
+
+_inlined_apis = [
```

Comment:
Is this _yet another_ place which has to be manually kept up-to-date?

If so, I strongly suggest that we update the `CONTRIBUTING.md` guide with a section which explains how docs ought to be written:
- docstrings on classes/functions are clear
- add its reference to the correct apidocs location in `docs/apidocs/*.rst`
- if its docs are inlined, also add it here

Anything else?

Reply:
> Is this yet another place which has to be manually kept up-to-date?

Nope, it is not necessary to ever update. This is only a one-time thing as we do the initial reorganization to inline APIs. It's to avoid breaking users/SEO familiar with the old structure.

In the future, you will directly inline functions so there won't be any need to set up a redirect.

Reply:
Ah I see. Thanks for clarifying :+1: